File: BankingTool.Model/Dto/RequestDto/User/GetUserInitialLoadRequest.cs
using BankingTool.Model.Dto.RequestDtos;

namespace BankingTool.Model.Dto.RequestDto.User
{
    public class GetUserInitialLoadRequestObject
    {
        public GetUserInitialLoadRequest Request { get; set; }
    }
    public class GetUserInitialLoadRequest
    {
        public GetUserInitialLoadRequestUser User { get; set; }
    }
    public class GetUserInitialLoadRequestUser : RequestNullableId
    {
    }

}

File: bankingtool.client/src/app/dto/request/user/get-user-initial-load-request.ts
import { RequestNullableId } from "../request-id";

export interface GetUserInitialLoadRequestObject {
  request: GetUserInitialLoadRequest;
}

export interface GetUserInitialLoadRequest {
  user: GetUserInitialLoadRequestUser;
}

export interface GetUserInitialLoadRequestUser extends RequestNullableId {

}
